### DISCUSSION TITLE
Feature Request: Minimize to tray icon

### DISCUSSION THREAD

**Discussion by ternite** [iaincollins/icarus#17] [Ideas & Suggestions]
Hi,

I'm using the fabulous functionality to access this app via browser on mobile device. In that case, on my PC there is still the Terminal Launcher Windows, which has to share precious space on my task bar with my shortcuts and an array of open apps.

It would be nice if minimizing the Terminal Launcher would place a small icon to the icon tray on the right. That's where in my opinion, background processes like the Terminal Launcher should be located.

And thanks for this great tool!

**Comment by iaincollins** [comment]
I agree - I think this is a good idea.

I full appreciate it's a bit annoying to have a launcher window that needs to be left open but went with it as was less worse than installing it as a Windows background service that would be invisible but chew up resources even when not running the game.

The only reason I haven't done this yet is not gotten around to it, but it's good to be reminded of.
